adafruit_rfm9x.py

```diff
@@ -243,21 +253,23 @@ def __set__(self, obj, val):
 
     def __init__(
         self,
-        spi,
-        cs,
-        reset,
-        frequency,
+        spi: SPI,
+        cs: DigitalInOut,
+        reset: DigitalInOut,
+        frequency: int,
         *,
-        preamble_length=8,
-        high_power=True,
-        baudrate=5000000,
-        agc=False,
-        crc=True
-    ):
+        preamble_length: int = 8,
+        high_power: bool = True,
+        baudrate: int = 5000000,
+        agc: bool = False,
+        crc: bool = True
+    ) -> None:
         self.high_power = high_power
         # Device support SPI mode 0 (polarity & phase = 0) up to a max of 10mhz.
         # Set Default Baudrate to 5MHz to avoid problems
-        self._device = spidev.SPIDevice(spi, cs, baudrate=baudrate, polarity=0, phase=0)
+        self._device: spidev.SPIDevice = spidev.SPIDevice(
```

Comment:
Ah just noticing this now. @FoamyGuy, this didn't seem to affect the build, and `mpy-cross` is able to compile the file just fine. We've had issues in the past with type annotating attributes though, so I wanted to get your insight.

Reply:
It may just be best to remove it then - no loose ends is better than few!

Other than than, it just seems to be type annotations, so if we can test with the hardware I think we're golden!